src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,84 @@
+import { Component } from 'react';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { withStyles } from '@material-ui/core/styles';
+import { arrayOf, object } from 'prop-types';
+import { camelCase } from 'change-case';
+import { memoizeWith } from 'ramda';
+import DataTable from '../DataTable';
+
+@withStyles(() => ({
+  summary: {
+    whiteSpace: 'nowrap',
+  },
+}))
+export default class BugsTable extends Component {
+  static propTypes = {
+    /**
+     * A list of objects to display. Each element in the list is represented
+     * by a row and each element's key-value pair represents a column.
+     */
+    items: arrayOf(object).isRequired,
+  };
+
+  state = {
+    sortBy: null,
+    sortDirection: null,
+  };
+
+  getTableData = memoizeWith(
+    ({ sortBy, sortDirection }) => `${sortBy}-${sortDirection}`,
+    ({ sortBy, sortDirection, items }) => {
+      const sortByProperty = camelCase(sortBy);
+
+      if (!items) {
```

Comment:
If `items` is a required array according to `propTypes`, then we don't need this if statement.